nextflow_schema.json

```diff
@@ -36,7 +31,7 @@
                     "description": "multiqc.zip folder containing the multiQC plots and report."
                 }
             },
-            "required": ["metadata", "model", "species", "project_summary", "multiqc"]
+            "required": ["metadata", "model", "project_summary", "multiqc"]
```

Comment:
```suggestion
            "required": ["metadata", "model", "project_summary", "multiqc", "gene_counts"]
```

Reply:
Also `gene_count` param description should be updated for the cases of `rsem` and `salmon`